LeafBot/LeafBot/Commands/HelpFormatter.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using DSharpPlus;
+using DSharpPlus.Entities;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Converters;
+using DSharpPlus.CommandsNext.Entities;
+
+namespace LeafBot.Commands
+{
+  // Custom formatter for any help commands
+  class HelpFormatter : BaseHelpFormatter
+  {
+    private DiscordEmbedBuilder EmbedBuilder { get; }
+    private DiscordEmoji HelpEmoji { get; }
+
+    public HelpFormatter(CommandContext ctx) : base(ctx)
+    {
+      this.EmbedBuilder = new DiscordEmbedBuilder();
+      this.HelpEmoji = DiscordEmoji.FromName(ctx.Client, ":books:");
+
+      EmbedBuilder.Color = DiscordColor.SpringGreen;
+    }
+    public override BaseHelpFormatter WithCommand(Command command)
+    {
+      EmbedBuilder.Title = $"{HelpEmoji} '{command.Name}'";
+
+      EmbedBuilder.AddField("Description", Formatter.Italic(command.Description));
+      EmbedBuilder.AddField("Aliases", string.Join(", ", command.Aliases.Select(x => Formatter.InlineCode(x))));
```

Comment:
need to handle edge case here: this will fail with a `ArgumentException` for `AddField` if there are no aliases

eg try it with `!help ping`

Reply:
Very good catch. I'll add a check

Reply:
Added fix so this shouldn't happen anymore